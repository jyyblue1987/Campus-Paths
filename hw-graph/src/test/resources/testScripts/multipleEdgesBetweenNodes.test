# To test if the graph can support multiple edges between two nodes

# Create a graph
CreateGraph A

# Add two nodes
AddNode A n1
AddNode A n2

# Add 6 edges
AddEdge A n1 n1 e11
AddEdge A n1 n2 e12
AddEdge A n2 n2 e22
AddEdge A n2 n1 e21
AddEdge A n2 n1 e21a
AddEdge A n1 n1 e11a

# Print out the nodes in the graph along with their outgoing edges
ListNodes A
ListChildren A n1
ListChildren A n2