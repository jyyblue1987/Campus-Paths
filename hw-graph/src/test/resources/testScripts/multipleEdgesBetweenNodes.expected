# To test if the graph can support multiple edges between two nodes

# Create a graph
created graph A

# Add two nodes
added node n1 to A
added node n2 to A

# Add 6 edges
added edge e11 from n1 to n1 in A
added edge e12 from n1 to n2 in A
added edge e22 from n2 to n2 in A
added edge e21 from n2 to n1 in A
added edge e21a from n2 to n1 in A
added edge e11a from n1 to n1 in A

# Print out the nodes in the graph along with their outgoing edges
A contains: n1 n2
the children of n1 in A are: n1(e11) n2(e12) n1(e11a)
the children of n2 in A are: n1(e21) n1(e21a) n2(e22)